src/generators/api-dns/index.ts

```diff
@@ -1,30 +1,29 @@
-const { BaseGenerator, processDestinationPath, domainName, listZones, listApis } = require('../../common');
+import { BaseGenerator, listApis, listZones } from '../../common';
 
 class ApiDnsGenerator extends BaseGenerator {
-
   constructor(args, opts) {
     super(args, opts);
 
-    this._input({ name: "api", type: 'list', choices: listApis(this.destinationRoot()) });
-    this._input({ name: "zone", type: 'list', choices: listZones(this.destinationRoot()) });
+    this._input({ name: 'api', type: 'list', choices: listApis(this.destinationRoot()) });
+    this._input({ name: 'zone', type: 'list', choices: listZones(this.destinationRoot()) });
     this._input({ name: 'subdomain', type: 'input' });
     this._input({ name: 'base_path', type: 'input' });
   }
 
   async create() {
     let answers = await this._prompt();
     const { zone, subdomain } = answers;
-    answers.safe_name = subdomain.replace(/\./g, "-") + "-" + zone;
+    answers.safe_name = subdomain.replace(/\./g, '-') + '-' + zone;
 
-    await this.fs.copyTplAsync(
+    this.fs.copyTpl(
```

Comment:
just a curiosity - I'm sure it changes nothing - but why switch from async?

Reply:
according to the typing, fs.copyTplAsync doesn't exist for the generator class. Previous there were some imports of `const fs = require('fs/promise')`

We are doing awaits on the async copy, which makes it synchronous anyway. so using sync copy is basically the same